app/controllers/book_discussions_controller.rb

```diff
@@ -0,0 +1,11 @@
+class BookDiscussionsController < DiscussionsController
+  private
+
+  def set_debatable
+    @debatable = Organization.current.book
+  end
+
+  def set_current_discussions
+    @discussions = Discussion.all
```

Comment:
I suggest creating a method that just returns `Discussions.all` , e.g. `current_content_discussions`